samples/drivers/can/sample.yaml

```diff
@@ -9,3 +9,6 @@ tests:
       type: one_line
       regex:
         - "Counter received: (.*)"
+  sample.drivers.can.dfrobot_can_bus_v2_0.ssd1306_128x64:
```

Comment:
`.ssd1306_128x64` does not seem to be quite right.

Reply:
Ah, sorry, copypaste leftover, fixed.